edge-modules/api-proxy-module/src/monitors/config_parser.rs

```diff
@@ -6,7 +6,11 @@ use regex::{Captures, Regex};
 
 use crate::token_service::token_server;
 
-const PROXY_CONFIG_DEFAULT_VALUES: &[(&str, &str)] = &[("NGINX_DEFAULT_PORT", "443")];
+const PROXY_CONFIG_DEFAULT_VALUES: &[(&str, &str)] = &[
+    ("NGINX_DEFAULT_PORT", "443"),
+    ("NGINX_DEFAULT_TLS", "TLSv1 TLSv1.1 TLSv1.2"),
+    ("NGINX_DEFAULT_CIPHERS", "HIGH:!aNULL:!MD5"),
```

Comment:
So, what are the default ciphers? (can we just use the nginx default list?)

Reply:
Yes, those are the default from nginx default list. There is not "default" variable I can use. I see another way to do it with ifdef, I'll do something else.